tomviz/ModuleScaleCube.h

```diff
@@ -73,6 +73,7 @@ protected slots:
   ModuleScaleCubeWidget* m_controllers = nullptr;
   unsigned long m_observedPositionId;
   unsigned long m_observedSideLengthId;
+  bool m_annotationVisibility;
```

Comment:
You can use C++11 initialization inline here, i.e.

    bool m_annotationVisibility = false;